packages/@uppy/provider-views/src/ProviderView/AuthView.js

```diff
@@ -1,26 +1,81 @@
 const { h } = require('preact')
 
+function GoogleIcon () {
```

Comment:
Doesn't Google host the SVG themselves? It might be safer to get it from their server if they ever update the icon – and the user as to connect to it anyway in order to be authenticated.

Reply:
Unfortunately not. Ideally they would provide the styles as well if you want to be 100% brand compliant all the time, but that's not happening either.